### PR TITLE
[v6] Use basset block directive in FREE fields

### PR DESCRIPTION
**Pull request by tabacitu**
### BEFORE - What was wrong? What was happening before this PR?

Some FREE fields were still use `@loadOnce` instead of the new `@bassetBlock`

### AFTER - What is happening after this PR?

All fields are using `@bassetBlock` instead, which is not only loading it once... but also MOVING that bit of CSS/JS to a file, and afterwards load it from there. This has two benefits:
- (1) it cleans up the output (no more inline CSS and JS because of the fields)
- (2) caching; because it's a JS/CSS file, the browser can cache it, so it SHOULD be faster (haven't benchmarked);

Just look at how clean the directory looks, with all the CSS and JS that used to be inline:
CleanShot 2023-02-21 at 10 01 20@2x

But wait till we do the same with the PRO repo 🤯 Over there, all fields have some sort of inline JS or CSS. So all of that will get moved to files.

### How did you achieve that, in technical terms?

Manually replaced ALMOST all ocurrences.

### Is it a breaking change?

For v5? Yes.
For v6? Not if you're using the `bassets` branch of DigitallyHappy/Assets, no.

### CODE REVIEW

File: src/resources/views/crud/buttons/delete.blade.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 {{-- Button Javascript --}}
 {{-- - used right away in AJAX operations (ex: List) --}}
 {{-- - pushed to the end of the page, after jQuery is loaded, for non-AJAX operations (ex: Show) --}}
+{{-- - cannot use bassetBlock here, because of inline PHP that we WANT rendered (ability to change lang) --}}
```
Comment (tabacitu) [inline src/resources/views/crud/buttons/delete.blade.php:8]: Ok so here's the deal. Here's the problem. This code block... has PHP inside it. And not any kind of PHP... but translations... So YES, we could use `@bassetBlock` here, and that entire script would get internalized... but it would get insternalized with EN translations.

If the developer changes the `app()->getLocale()` to FR, then... at the second call... that file is already internalized... and the messages are in ENGLISH, unfortunately.

Comment (tabacitu) [reply src/resources/views/crud/buttons/delete.blade.php]: Later edit. Ok soo... this limitation... the one that I'm talking about... it's not quite a limitation... as much as it is PROOF of how FUCKING AWESOME bassets is. Because instead of saying "_no this can't be done_" what I've done is... created a file for each language 🤯

Because YES, using bassets we can do 
```php
@bassetBlock('backpack/crud/buttons/delete-button-'.app()->getLocale().'.js')
```

Which has downsides:
- this will NOT get internalized using `php artisan basset:internalize`

But it also has upsides:
- we now have that bit of JS separated, for each individual language the admin panel might use:

CleanShot 2023-02-21 at 09 56 35@2x

So in the end... I think this is a pretty freakin' good solution. In fact... as I said... I think it's PROOF of how freakin' awesome 🐶 is.

Comment (tabacitu) [reply src/resources/views/crud/buttons/delete.blade.php]: So no, this usage does NOT highlight we should keep `@loadOnce` 🎉
But there might be other usages... that do 😔

**Comment by tabacitu** [comment]
TLDR: Backpack/CRUD is not free of `@loadOnce`. So it's no longer dependant on `digitallyhappy/assets`.

File: src/resources/views/crud/buttons/delete.blade.php (hunk 1 above)
Comment (tabacitu) [reply src/resources/views/crud/buttons/delete.blade.php]: And sure enough, some PRO fields have posed big problems: https://github.com/Laravel-Backpack/PRO/issues/128
